torch/csrc/jit/script/compiler.cpp

```diff
@@ -3402,6 +3403,8 @@ void runCleanupPasses(std::shared_ptr<Graph>& to_clean) {
   // remove any uses of tuples that we inserted that are not needed
   LowerSimpleTuples(to_clean);
   ConstantPooling(to_clean);
+  // run constant propagation only on non-mutable types so we do not jitter
```

Comment:
Nit: the comment reads as if we run constant prop to avoid jitter, but I believe that you meant that we're running const-prop **only on non-mutable types** in order for const-prop to not introduce jitter. I hope I managed to explain what I meant :)